src/pkg/caendr/caendr/services/cloud/analytics.py

```diff
@@ -37,6 +45,7 @@ def get_weekly_visits(SERVICE):
       'reportRequests': [{
         'viewId': '117392266',
```

Comment:
not part of the PR, but avoid having `magic numbers`, try to create constants and reference them, or consider moving these to part of the config (ENV VARS) for the app. 
E.g.: GOOGLE_ANALYTICS_VIEW_ID: 117392266